tests/src/ViewModel/TeaserTest.php

```diff
@@ -50,8 +50,7 @@ public function it_can_load_main()
             null,
             TeaserFooter::forArticle(
                 $this->metaFromData($data['footer']['meta']),
-                $data['footer']['formats']['html'],
-                $data['footer']['formats']['pdf']
+                $data['footer']['formats']['list']
```

Comment:
got that this indirectly comes from the `pattern-library`'s JSON files, good reuse

Reply:
Not really, from the JSON files in this project. Still good and more customizable probably.